src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -66,6 +70,24 @@ def __init__(self, job_id, initial_params, area, should_export_plots):
                 ConstSettings.GeneralSettings.EXPORT_ENERGY_TRADE_PROFILE_HR:
             self.area_market_stocks_stats = OfferBidTradeGraphStats()
 
+    @staticmethod
+    def _structure_results_from_area_object(target_area):
+        area_dict = dict()
+        area_dict['name'] = target_area.name
+        area_dict['uuid'] = target_area.uuid
+        area_dict['strategy'] = str(target_area.strategy.__class__.__name__)
+        area_dict['appliance'] = str(target_area.strategy.__class__.__name__)
```

Comment:
There are a couple of points that would be nice to have here. First, appliance is not needed, and probably won't in the future, since we will get rid of the appliance mechanism, therefore I would recommend to remove it. Second, if the strategy is None (which is the case for area objects) the clause `target_area.strategy.__class__` will probably not fail, but will return the class type of the `None` object. Not that good for using it in the API which is transmitted across our services. What I would recommend to not use the term strategy, but `type` instead. That way, if strategy is None, you can assign a type `Area`.